README.md

```diff
@@ -1,2 +1,64 @@
 # wg-meta
-An apporach to add metadata to the main wireguard config
+An approach to add metadata to the main wireguard config, written in Perl.
+
+## Installation
+This project supports automake
+```shell script
+./bootstrap
```

Comment:
bootstrap is only required when downloading from github ... the tar.gz file does not have to be bootrapped